app/utils/dayjs.ts

```diff
@@ -1,5 +1,5 @@
 import dayjs from "dayjs"
-import dayJsutcPlugin from "dayjs/plugin/utc"
+import dayJsutcPlugin from "dayjs/plugin/utc.js"
```

Comment:
this is needed?

Reply:
Yup 

![image](https://github.com/pokt-foundation/portal-platform/assets/38038348/74404475-0e26-4d1a-82a6-2771d49126eb)